example/handwritten-src/swift/TXSTextboxListenerDebugableImpl.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TXSTextboxListenerImpl.swift
+//  TextSort
+//
+//  Created by Bruno Coelho on 10/03/2017.
+//  Copyright © 2017 Dropbox, Inc. All rights reserved.
+//
+
+import UIKit
+
+@objc class TXSTextboxListenerDebugableImpl : NSObject, TXSTextboxListener {
+    private var textView_: UITextView
+    
+    @objc(initWithUITextView:)
+    init (textView: UITextView) {
+        textView_ = textView
+    }
+    
```

Comment:
you could add
```
    @available(*, unavailable)
    init() {
        fatalError("Unsupported")
    }
```
to be explicit the default `NSObject` initializer is not available.